gordo_components/server/views/base.py

```diff
@@ -304,17 +304,41 @@ def multi_lvl_column_dataframe_to_dict(df: pd.DataFrame) -> typing.List[dict]:
         Convert a dataframe which has a pandas.MultiIndex as columns into a dict
         where each key is the top level column name, and the value is the array
         of columns under the top level name.
+
+        Notes
```

Comment:
I think this notes section should be for the users of the function, not the future programmers. I.e. i like the comment, but I want it below with the actual code